Khan/wonder-blocks
# Add MediaLayout documentation to storybook

## Summary:
Using the documentation in Styleguidist as a foundation, add
MediaLayout documentation to storybook. Note that this component is being
deprecated.

Issue: https://khanacademy.atlassian.net/browse/WB-1139

## Test plan:
- Confirm that all the stories change background colors when resizing the window.
- Confirm there are no typos in the documentation.
- Read through the code snippets and confirm they are readable.
